src/ext/ddtrace.c

```diff
@@ -377,7 +380,7 @@ static PHP_RINIT_FUNCTION(ddtrace) {
     // Reset compile time after request init hook has compiled
     ddtrace_compile_time_reset(TSRMLS_C);
 
-    DDTRACE_G(traces_group_id) = ddtrace_coms_next_group_id();
+    _ddtrace_prepare_for_new_trace(TSRMLS_C);
```

Comment:
To match my other suggestions:
```suggestion
    dd_prepare_for_new_trace(TSRMLS_C);
```